lib/chef/knife/ec_restore.rb

```diff
@@ -19,6 +19,12 @@ class EcRestore < Chef::Knife
         :long => "--skip-users",
         :description => "Skip restoring users"
 
+      option :skip_ids,
```

Comment:
I opted to just duplicate this option from ec_key_import.rb to ec_restore.rb. It would be ideal if we could stick the option in a common base, but ec_restore.rb includes ec_base.rb and ec_key_import.rb includes ec_key_import.rb. I would have to restructure a large chunk of knife-ec-backup to avoid duplicating this option. Since there are other duplicated options between code paths, I'm going to favor the approach of least change, and kick this particular can down the road.